merkle_tree_wasm/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+name = "merkle_tree_wasm"
```

Comment:
the whole branch needs to be rebased.
E.g You renamed this to `merkle-tree-wasm` in #275

Reply:
Done.